jwst/straylight/straylight_step.py

```diff
@@ -14,41 +14,59 @@ class StraylightStep (Step):
 
     class_alias = "straylight"
 
-    reference_file_types = ['mrsxartcorr']
+    spec = """
+        clean_showers = boolean(default=False)  # Clean up straylight from residual cosmic ray showers
+        shower_plane = integer(default=3)  # Throughput plane for identifying inter-slice regions
+        shower_x_stddev = float(default=18) # X standard deviation for shower model
+        shower_y_stddev = float(default=5) # Y standard deviation for shower model
+        shower_low_reject = float(default=0.1) # Low percentile of pixels to reject
+        shower_high_reject = float(default=99.9) # High percentile of pixels to reject
+    """
```

Comment:
Can you please add this note for the style checker at the end of the spec?  We just added these to all other existing spec definitions yesterday to allow long strings in the parameter help comments.  Since this definition is new, it needs to be added here.

```suggestion
    """  # noqa: E501
```